src/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<script src="https://kit.fontawesome.com/2b1476549e.js" crossorigin="anonymous"></script>
+    <title>To do</title>
+</head>
+<body>
```

Comment:
- [ ] The styling of your app design does not match the provided [design template](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Kindly ensure your design follows the design template in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg).

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/154248902-1b4d2540-740c-4d95-aeb6-5c9febfbbeb5.png)

**The original design:**
![image](https://user-images.githubusercontent.com/12745474/134218832-b8df2741-aec5-4c3d-9cc9-423a49f35c8b.png)

Reply:
@daudi13 
Sure..I've messaged you on slack, kindly send the link in the thread!

Reply:
@daudi13 
As per our discussion, some commits were made to this PR after your project has been submitted. Kindly ensure that commit/ updates are done only after review is completed for every submitted project! 

Happy coding! 😃